user/contactmethod/store.go

```diff
@@ -306,12 +229,23 @@ func (s *Store) FindOneTx(ctx context.Context, tx *sql.Tx, id string) (*ContactM
 		return nil, err
 	}
 
-	var c ContactMethod
-	row := wrapTx(ctx, tx, s.findOneUpd).QueryRowContext(ctx, id)
-	err = row.Scan(&c.ID, &c.Name, &c.Type, &c.Value, &c.Disabled, &c.UserID, &c.lastTestVerifyAt, &c.StatusUpdates, &c.Pending)
+	row, err := gadb.New(s.db).FindOneUpdateContactMethod(ctx, uuid.MustParse(id))
```

Comment:
replace line 227 with:
`err, methodUUID := validate.ParseUUID("ContactMethodID", id)`
and then you can do this:
```suggestion
	row, err := gadb.New(s.db).FindOneUpdateContactMethod(ctx, methodUUID)
```